src/Plugins/SimplnxCore/src/SimplnxCore/Filters/IterativeClosestPointFilter.cpp

```diff
@@ -22,6 +22,7 @@ constexpr int32 k_MissingMovingVertex = -4500;
 constexpr int32 k_MissingTargetVertex = -4501;
 constexpr int32 k_BadNumIterations = -4502;
 constexpr int32 k_MissingVertices = -4503;
+constexpr int32 k_EmpyVertices = -4506;
```

Comment:
```suggestion
constexpr int32 k_EmptyVertices = -4504;
```
Typo and Error Code increased to match the ordering of the above codes